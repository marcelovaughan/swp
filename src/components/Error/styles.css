.error-wrapper{
    height: 100vh;
    display: flex;
    vertical-align: middle;
    justify-content: center;
    align-items: center; 
  }
  
  .error {
    margin: 0px auto;
    width: 78%;
    display: inline-block;
    padding: 15px;
  }
  
  .error h2 {
    color: #ffea00;
    text-align: center;
    font-size: 18px;
  }
  
  .error p {
    color: #fff;
    text-align: center;
    text-transform: uppercase;
  }
  
  .error-img {
    background: url('../../assets/img/Death_Star_II.png');
    height: 250px;
    width: 250px;
    background-size: contain;
    text-align: center;
    color: #fff;
    font-size: 62px;
    display: flex;
    margin: 20px auto;  
  }