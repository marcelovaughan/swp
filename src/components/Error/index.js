import React from 'react';
import Header from '../Header';

import './styles.css';

export default function Error() {
    return (
        <div className="error-wrapper">
            <Header/>
            <div className="error">
                <div className="error-img"></div>
                    <div className="error-msg">
                    <h2>ESTA PÁGINA NÃO ESTÁ ARMADA E OPERACIONAL.</h2>
                    <p>Perdemos o contato com nossa fonte de dados.</p>
                </div>        
            </div>
        </div>
    );
}