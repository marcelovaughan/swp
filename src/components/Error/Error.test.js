import React from "react"
import { configure, mount, shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
import Error from "./index";

configure({ adapter: new Adapter() });

describe("Error Render", () => {
    it("Render the component", () => {
        const component = mount(<Error />);
        expect(component.debug()).toMatchSnapshot();
    });
});
