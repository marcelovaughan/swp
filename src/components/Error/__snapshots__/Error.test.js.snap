// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Render Render the component 1`] = `
"<Error>
  <div className=\\"error-wrapper\\">
    <Header>
      <header>
        <h1 className=\\"logo\\">
          StarWars - Planets
        </h1>
        <h2>
          PLANETS
        </h2>
      </header>
    </Header>
    <div className=\\"error\\">
      <div className=\\"error-img\\" />
      <div className=\\"error-msg\\">
        <h2>
          ESTA PÁGINA NÃO ESTÁ ARMADA E OPERACIONAL.
        </h2>
        <p>
          Perdemos o contato com nossa fonte de dados.
        </p>
      </div>
    </div>
  </div>
</Error>"
`;
